ChangeLog

```diff
@@ -22,6 +22,16 @@ Security
    * Wipe sensitive buffers on the stack in the CTR_DRBG and HMAC_DRBG
      modules.
 
+Features
+   * Add a new config.h option of MBEDTLS_CHECK_PARAMS that enables additional
```

Comment:
Minor: Remove `of`.

Reply:
I don't think you can just remove `of` without rephrasing the rest of the sentence. It just doesn't work.

Reply:
Why? "Add a new config.h option MBEDTLS_CHECK_PARAMS that enables ..." is perfectly fine, isn't it?